OpenEdXMobile/res/values/styles.xml

```diff
@@ -559,6 +559,13 @@
         <item name="android:windowExitAnimation">@anim/slide_down</item>
     </style>
 
+    <style name="CourseTabItem">
+        <item name="android:textSize">@dimen/edx_small</item>
+        <item name="lineHeight">@dimen/widget_margin_double</item>
```

Comment:
looks high as compared to [Figma](https://www.figma.com/file/Vaw3lfBXmOhTLKFnBedqE1/App-Navigation-Improvements?node-id=1492%3A0&t=OnlRSu6EvIuqmyyR-0) designs